### ISSUE TITLE
Does it make sense to implement ORPHAcodes with regard of the introduction of ICD11? Will ORPHAcodes be listed in ICD11?

### ISSUE THREAD

**Issue by OrphanetDE** [OD4RD/Main-Help-Desk#18]
Does it make sense to implement ORPHAcodes with regard of the introduction of ICD11? Will ORPHAcodes be listed in ICD11?

**Comment by Marlies99** [comment]
Hi

I'll take the liberty of sharing some of my findings and thoughts here and make a bit of conversation, as a response from the Github team can take a long time. We will see what happens ;-).

According to the WHO, ICD 11 contains about 5500 rare diseases:
https://www.who.int/standards/classifications/frequently-asked-questions/rare-diseases

The nomenclature pack of orphadata version July 2022, contains, according to my calculation, 10675 clinical entities divided into 2488 groups of disorders, 6937 disorders and 1250 subtypes of disorders. In my opinion, orphadata is still more comprehensive and up-to-date. And in Switzerland, it will probably take a few more years before ICD 11 is introduced. Moreover, there is no obligation to code outpatients with ICD-codes in Switzerland (to many patients). This means that ICD coding currently almost only covers inpatients. And, the ICD coding is primarily intended for case billing. The underlying (rare) disease is not always coded as a result.

Cheers
Marlies

**Comment by JTorphanet** [comment]
Hello OrphanetDE and Marlies99, 
 
As of today, we have aligned about 2000 ORPHAcodes with ICD-11, corresponding to 32% of ORPHAcodes of which 20% with an exact relationship. We have started a close collaboration with WHO to introduce rare diseases with the best possible representation in ICD-11, but as you probably know, it is not an ultimate goal for ICD-11 to propose an international classification of rare diseases. Moreover, not all rare diseases in the ICD-11 will have a specific code, but the majority of rare diseases will be represented in the Foundation with an URI that will correspond to a larger ICD-11 code.
  At short and long term, ICD-11 is not intended to replace ORPHAcodes. ORPHAcodes will still be needed as complementary codes to retrieve specific information and make statistical analysis on rare diseases. More precise information about this topic will also be available when the alignment process between the two terminologies will be completed.

Feel free to consult the wikipage in the coming months, we will add some info about this collaboration !

Best, 
Julie
